src/main/java/co/norse/chat/controller/MessageController.java

```diff
@@ -0,0 +1,35 @@
+package co.norse.chat.controller;
+
+import co.norse.chat.model.Message;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Controller
+public class MessageController {
+    List<Message> allMessage = new ArrayList<>();
```

Comment:
rename to messages

Reply:
I did delete this field,its don`t exist

Reply:
I have deleted this field, it doesn't exist